NomaiVR/Hands/Hand.cs

```diff
@@ -10,8 +11,10 @@ namespace NomaiVR.Hands
 {
     public class Hand : MonoBehaviour
     {
+        public const float k_minimumPointDistance = 0.1f;
```

Comment:
Naming convention now is pretty much just camelCase for private, PascalCase for public, no underscores or other identifiers. Doesn't matter if it's const, static, etc.